develop/parachains/deployment/coretime-renewal.md

```diff
@@ -0,0 +1,173 @@
+3. Upon successful payment, the system emits a `Renewed` event and secures the core for the next period
+4. If payment fails due to insufficient funds or other issues, the system emits an `AutoRenewalFailed` event
+
+Even if an auto-renewal attempt fails, the auto-renewal setting remains active for subsequent sales. This means once you've configured auto-renewal, the setting persists across multiple periods.
+
+There is a limit on the total number of auto-renewals allowed, specified at the runtime level as `T::MaxAutoRenewals`.
+
+To enable auto-renewal for your parachain, you'll need to configure several components as detailed in the following sections.
+
+### Set Up HRMP Channel
+
+An Horizontal Relay-routed Message Passing (HRMP) channel must be open between your parachain and the Coretime System Chain before auto-renewal can be configured. 
+
+For instructions on establishing this connection, consult the [Opening HRMP Channels with System Parachains](/tutorials/interoperability/xcm-channels/para-to-system/){target=\_blank} guide.
+
+### Fund Sovereign Account
+
+The [sovereign account](https://github.com/polkadot-fellows/xcm-format/blob/10726875bd3016c5e528c85ed6e82415e4b847d7/README.md?plain=1#L50){target=\_blank} of your parachain on the Coretime chain needs adequate funding to cover both XCM transaction fees and the recurring coretime renewal payments.
+
+To determine your parachain's sovereign account address, you can:
+
+- Use the "Para ID" to Address section in [Substrate Utilities](https://www.shawntabrizi.com/substrate-js-utilities/){target=\_blank} with the **Sibling** option selected
+
+- Calculate it manually:
+
+    1. Identify the appropriate prefix:
+
+        - For sibling chains: `0x7369626c` (decodes to `b"sibl"`)
+         
+    2. Encode your parachain ID as a u32 [SCALE](https://docs.polkadot.com/polkadot-protocol/basics/data-encoding/#data-types){target=\_blank} value:
+
+        - For parachain 2000, this would be `d0070000`
+
+    3. Combine the prefix with the encoded ID to form the sovereign account address:
+
+        - **Hex** - `0x7369626cd0070000000000000000000000000000000000000000000000000000`
+        - **SS58 format** - `5Eg2fntJ27qsari4FGrGhrMqKFDRnkNSR6UshkZYBGXmSuC8`
+
+### Auto-Renewal Configuration Extrinsics
+
+The Coretime chain provides two primary extrinsics for managing the auto-renewal functionality:
+
+- `enable_auto_renew(core, task, workload_end_hint)` - use this extrinsic to activate automatic renewals for a specific core. This transaction must originate from the sovereign account of the parachain task
+
+    **Parameters:**
+
+    - **`core`**: the core currently assigned to the task
+    - **`task`**: the task for which auto-renewal is being enabled
+    - **`workload_end_hint`**: used when the core is not expiring in the current bulk period (e.g., due to a lease). This allows you to enable auto-renewal in advance, instead of needing to wait for the sale in which your core will expire
```

Comment:
The recommendation is to always use this to avoid confusion. If the coretime is expiring in the current sale period, use the last timeslice number of current sale period. If it is expiring at the end of next sale period (if you purchased coretime already), then use the last timeslice number of next sale period. If there is a lease, then the timeslice when the lease ends.

Reply:
Suggestions applied